packages/synapse-react-client/src/components/EntityHeaderTable/useEntityHeaderTableState.ts

```diff
@@ -15,6 +15,10 @@ export function useEntityHeaderTableState(
   const [newEntityIDs, _setNewEntityIDs] = useState<string>('')
   const [parseErrors, setParseErrors] = useState<string[]>([])
 
+  useEffect(() => {
+    _setRefsInState(cloneDeep(references))
+  }, [references])
```

Comment:
Just want a comment here for clarity
```suggestion
  /* Reset state if the `references` argument changes */
  useEffect(() => {
    _setRefsInState(cloneDeep(references))
  }, [references])
```